apps/public-reference/pages/_error.jsx

```diff
@@ -0,0 +1,27 @@
+import Layout from "../layouts/application"
+import Head from "next/head"
+import t from "@bloom-housing/ui-components/src/helpers/translator"
+import Hero from "@bloom-housing/ui-components/src/headers/Hero/Hero"
+import MarkdownSection from "@bloom-housing/ui-components/src/sections/markdown_section"
+import PageContent from "../page_content/homepage.mdx"
+
+export default () => {
+  const pageTitle = <>Page Not Found</>
+
+  return (
+    <Layout>
+      <Head>
+        <title>{pageTitle}</title>
+      </Head>
+      <Hero title={pageTitle} buttonTitle={t("welcome.seeRentalListings")} buttonLink="/listings">
+        Uh oh, we can’t seem to find the page you’re looking for. Try going back to the previous
```

Comment:
Are we hard-coding the text here because we're worried about not being able to resolve translations, or can we make this a localized string too?

Reply:
@bk3c Good catch, I totally spaced. Moved those messages to the locale file now.